tests/commands/firewall.py

```diff
@@ -124,5 +125,12 @@ def runTest(self):
         self.assert_parse("firewall --service=mdns --remove-service=dhcpv6-client --remove-service=ssh",
                           "firewall --enabled --service=mdns --remove-service=dhcpv6-client,ssh\n")
 
+class F28_TestCase(F20_TestCase):
+    def runTest(self):
+        F20_TestCase.runTest(self)
+
+        # use-system-defaults
+        self.assert_parse("firewall --use-system-defaults\n")
+
```

Comment:
I'd like to see an additional test for when you use `--use-system-defaults` and some other option, to verify that part works.  That could also help out with the other comment I left.

Reply:
passing something like `firewall --use-system-defaults  --enabled --service=ssh,smtp,ftp ` probably doesn't make sense, though. the `--use-system-defaults` is literally like a short circuit that says 'do nothing'. Do you agree? What are you looking for here?

Reply:
If it doesn't make sense than it should be an exception.

Reply:
That's one possibility - yeah.  An exception could be raised.

I know it doesn't make sense.  What I'm looking for is a test to make sure that the thing that doesn't make sense is being handled correctly.

Reply:
ok, so you want me to raise an exception when someone provides `--use-system-defaults` and another option?

Reply:
I looked into this quite a bit today. I really tried to figure out how I could use argparse [mutual exclusion](https://docs.python.org/2/library/argparse.html#mutual-exclusion) to achieve this goal but unfortunately it seems the way things are wired up in anaconda (i.e. the overiding of the `add_argument` function) does not lend itself well to this. Do you know of any other examples where exclusivity is required in anaconda?

Within this very same module one can pass `--disable --enable` and anaconda will parse it fine.

Reply:
bump